### PR TITLE
Chinese - Chapter 0,2finished 

### PR DESCRIPTION
**Pull request by 1375626371**
Hello, we have finished the translation of chapter zero and chapter two, please check. Among them, sections 1, 3, 5, and 7 of the chapter 2 were completed by @petrichor1122 , sections 2, 4, and 6 of the chapter 2  were completed by @zhlhyx , section 8 of the chapter 2  and chapter 0 were completed by me.

### CODE REVIEW

**Comment by lewtun** [comment]
Hi @1375626371 did you accidentally close this PR? I'm happy to re-open and review it if you wish :) Just FYI that we now support `zh-CN` as the language code for the website, so I suggest renaming your folder as I did here: https://github.com/huggingface/course/pull/119

**Comment by 1375626371** [comment]
> Hi @1375626371 did you accidentally close this PR? I'm happy to re-open and review it if you wish :) Just FYI that we now support `zh-CN` as the language code for the website, so I suggest renaming your folder as I did here: #119
Okay, I understand. I will rename the folder in the next commit. The reason I closed this PR is that we found some errors during the code inspection. In order to ensure the quality, we have to re-check our translations. When the errors are corrected, we will submit them again.

**Comment by lewtun** [comment]
> Okay, I understand. I will rename the folder in the next commit. The reason I closed this PR is that we found some errors during the code inspection. In order to ensure the quality, we have to re-check our translations. When the errors are corrected, we will re-open them again.

Sounds great! Looking forward to it 🤗

**Comment by 1375626371** [comment]
> > Okay, I understand. I will rename the folder in the next commit. The reason I closed this PR is that we found some errors during the code inspection. In order to ensure the quality, we have to re-check our translations. When the errors are corrected, we will re-open them again.
> 
> Sounds great! Looking forward to it 🤗

Hello, we have completed the inspection, please help us open this PR. Thank you！

File: chapters/zh-CN/chapter2/3.mdx [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+{:else}
+```py
+from transformers import TFBertModel
+
+model = TFBertModel.from_pretrained("bert-base-cased")
+```
+
+正如您之前看到的，我们可以用等效的AutoModel类替换Bert模型。从现在开始，我们将这样做，因为这会产生检查点不可知的代码；如果您的代码适用于一个检查点，那么它应该与另一个检查点无缝地工作。即使体系结构不同，这也适用，只要检查点是针对类似任务（例如，情绪分析任务）训练的。
+{/if}
```
Comment (lewtun) [inline chapters/zh-CN/chapter2/3.mdx:144]: For some reason, we need a space here to build the docs. Can you please try this?

```suggestion

{/if}
```

Comment (lewtun) [reply chapters/zh-CN/chapter2/3.mdx:144]: Sounds great!

**Comment by 1375626371** [comment]
I run `make style` on local. But it still fails the quality check, what should I do?

**Comment by lewtun** [comment]
> I run `make style` on local before commit. But it still fails the quality check, what should I do?

I fixed the formatting issues and pushed a fix :) 

Since the CI passes, I think this is good to go! Thank you for the care and detail you've taken with this translation 🤗 !

**Review by lewtun — APPROVED** [review]

**Pull request merged**